### PR TITLE
test_cp_mgmt_host module

### PR DESCRIPTION
**Pull request by chkp-amiads**
##### SUMMARY

Checkpoint test_cp_mgmt_host module

##### ISSUE TYPE

- New Test Module Pull Request

##### COMPONENT NAME

checkpoint
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by gundalow** [comment]
```
test_cp_mgmt_host.py:0:0: missing: from __future__ import (absolute_import, division, print_function)
test_cp_mgmt_host.py:0:0: missing: __metaclass__ = type
```

File: test/units/modules/network/check_point/test_cp_mgmt_host.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+from units.modules.utils import set_module_args, exit_json, fail_json, AnsibleFailJson, AnsibleExitJson
+
+from ansible.module_utils import basic
+from ansible.module_utils.network.checkpoint.checkpoint import api_call
```
Comment (danielmellado) [inline test/units/modules/network/check_point/test_cp_mgmt_host.py:26]: this is not used and should be deleted

File: test/units/modules/network/check_point/test_cp_mgmt_host.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+from units.modules.utils import set_module_args, exit_json, fail_json, AnsibleFailJson, AnsibleExitJson
```
Comment (danielmellado) [inline test/units/modules/network/check_point/test_cp_mgmt_host.py:23]: AnsibleFailJson is not used and shouldn't be imported

**Review by danielmellado — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
Hi @chkp-amiads, Thank you very much for your interest in Ansible. This plugin/module is no longer maintained in this repository and has been migrated to https://github.com/CheckPointSW/CheckPointAnsibleMgmtCollection<br><br>If you have further questions please stop by IRC or the mailing list:<br><br>* IRC: #ansible on irc.freenode.net<br>* mailing list: https://groups.google.com/forum/#!forum/ansible-project<br>

**Pull request closed without merging**